src/main.rs

```diff
@@ -45,7 +45,7 @@ pub struct Opt {
 pub fn main() {
     let opt = Parser::parse();
     let exit_code = match run_opt(&opt) {
-        Ok(pass) => {
+        Ok((pass, _)) => {
```

Comment:
As per comment on line 62, this can become a 3-arm match where `Ok(Some(_))` returns 0, `Ok(None)` returns 1, and `Err(_)` returns 2.